rules/windows/defense_evasion_suspicious_scrobj_load.toml

```diff
@@ -13,35 +13,46 @@ from = "now-9m"
 index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
-name = "Windows Suspicious Script Object Execution"
-risk_score = 21
+name = "Suspicious Script Object Execution"
+risk_score = 47
 rule_id = "4ed678a9-3a4f-41fb-9fea-f85a6e0a0dff"
 severity = "medium"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
 type = "eql"
 
 query = '''
-/* add winlogbeat-* when process.code_signature.* fields are populated */
-
 sequence by process.entity_id with maxspan = 2m
-  [process where event.type in ("start", "process_started") and
-     /* uncomment once in winlogbeat */
-     /* process.code_signature.subject_name : "Microsoft Corporation" and process.code_signature.trusted : true and */
-     not process.name : (
-       "cscript.exe",
-       "iexplore.exe",
-       "MicrosoftEdge.exe",
-       "msiexec.exe",
-       "smartscreen.exe",
-       "taskhostw.exe",
-       "w3wp.exe",
-       "wscript.exe")]
+  [process where event.type == "start" and
+     (process.code_signature.subject_name in ("Microsoft Corporation", "Microsoft Windows") and
+      process.code_signature.trusted == true) and
```

Comment:
are the `code_signature` fields logged in winlogbeat now? If not, these rules will no longer work with `"winlogbeat-*",` or `"logs-windows.*"`.

Reply:
the rule description says "loaded into unusual Microsoft processe ... " without code sign check it will alert on non MS related processes. I will put back the comment for now.

Reply:
I remember the discussion on this one and the hope was to have signature information in winlog (which I don't think will be the case). So we can leave the signature in, but would need to remove the indexes (and the rule could only work on endpoint - which is not necessarily a bad thing since it would be a more accurate rule)

Reply:
winlogbeat index removed and codesig check enabled